Documentation/Signoffs/Device_Power.md

```diff
@@ -0,0 +1,189 @@
+# Device Power Subsystem
+
+**Function:**
+
+![Function](../Images/Device_Power/Conceptual2.png)
+
+Figure 1: Device Power Subsystem
+
+The goal of this subsystem is to convert AC power from the wall outlet to DC power, and then distribute that power to the different device systems.  
+
+**Constraints:**
+
+| NO. | Constraint                                                          | Origin           |
+|-----|---------------------------------------------------------------------|------------------|
+| 1   | The power system shall be connected to the emergency stop           |Conceptual Design |
+| 2   | The power system shall power the main/processor unit                |Conceptual Design |
```

Comment:
What does that mean?

Reply:
changed these to reflect the new system being included and made these more precise